### ISSUE TITLE
periodically freezing tightly when watching media content

### ISSUE THREAD

**Issue by stepan-neretin7** [linuxmint/cinnamon#10201]
```
 * Cinnamon version (cinnamon --version) -> Cinnamon 4.8.6
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Linux Mint 20.1
 * Graphics hardware *and* driver used  [INXI](https://pastebin.com/zgsTCe6Q)
 * 32 or 64 bit -> 64
 * Attach [~/.xsession-errors](https://termbin.com/ke0q), or [/var/log/syslog](https://termbin.com/sx5c)
 ```

**Issue**

Periodically (not to say that it is straightforward very often), when watching media content, the laptop freezes tightly
And this only happens on Linux. There is a Windows on dual block. There are no such problems
**Steps to reproduce**

I dont know :C

**Expected behaviour**
To avoid freezing: D

https://drive.google.com/file/d/1jJe-EJkds_vQzfB9Npes2KsHDIS0Lw8h/view?usp=sharing - The video itself where you can see what is happening

https://drive.google.com/file/d/1yyMOFrAwh89dLrtIXl1j_qEw_7O5W1er/view?usp=sharing atop logs after reboot create from last period

https://imgur.com/a/adKbfnB - atop logs too

I have a driver installed nvidia-driver-465

**Comment by icarter09** [comment]
Does this occur every time you want any media content?

**Comment by stepan-neretin7** [comment]
No, periodically so to speak ... I myself can not even determine the frequency, but this, unfortunately, happens

**Comment by leigh123linux** [comment]
What are you using for media playback?, the info posted so far isn't helpful.

**Comment by stepan-neretin7** [comment]
> What are you using for media playback?, the info posted so far isn't helpful.

i am using firefox browser to play media

**Comment by stepan-neretin7** [comment]
please help me :C
